### DISCUSSION TITLE
mysql如何原生执行sql

### DISCUSSION THREAD

**Discussion by xiaozhuang-a** [hhyo/Archery#2387] [General]
比如mysql8.0加字段、drop 索引等场景，不需要走大表ddl，如何避免

**Comment by lanheader** [comment]
<img width="1436" alt="image" src="https://github.com/hhyo/Archery/assets/52522047/0b6cdc31-5568-4676-abe1-9c47b379dec8">
我是修改了这里的逻辑，8.0以后是有online ddl

**Reply by xiaozhuang-a** [reply]
8.0也并不是所有ddl类型都支持only change metadata，会导致从库延迟的。
